release-notes/9.0/preview/preview5/api-diff/Microsoft.NETCore.App/9.0-preview5_System.IO.Pipelines.md

```diff
@@ -0,0 +1,99 @@
+# System.IO.Pipelines
+
+``` diff
++namespace System.IO.Pipelines {
```

Comment:
Do we need to do anything special to indicate that these are existing APIs that have been part of the Microsoft.AspNetCore.App shared runtime since its inception and published as a NuGet even longer?

The only thing new about it is its inclusion in the Microsoft.NETCore.App shared runtime as well. The System.Text.Json changes look good though. @BrennanConroy

Reply:
Just let me know and I can remove them. Is it the case for all the APIs in this namespace?

Reply:
I can address this in a separate PR. I'll merge this as is.